service/view_building_coordinator.cc

```diff
@@ -253,10 +261,22 @@ future<> view_building_coordinator::build_view(vbc_state state) {
                 continue;
             }
 
+            auto muts = co_await maybe_mark_build_status_started(guard, state, views, host_id);
+            cmuts.insert(cmuts.end(), std::make_move_iterator(muts.begin()), std::make_move_iterator(muts.end()));
             future<> rpc = send_task(target, *state.currently_processed_base_table, range, std::move(views));
             _rpc_handlers.insert({target, std::move(rpc)});
         }
     }
+
+    // TO CONSIDER: 
+    // What if this fails because of group0_concurrent_modification? Should we abort rpc calls then? 
+    // Or drop the guard and retry to commit the changes again?
```

Comment:
How about modifying the view_build_status_v2 table in `update_coordinator_state()`? You can first mark the view build as started, and only then - in the `build_view` function - spawn the necessary fibers for tracking the RPCs. I think this order would make a bit more sense rather than allowing the RPCs to be sent before the view is marked as being built.